### ISSUE TITLE
AutoAugment Question

### ISSUE THREAD

**Issue by RaivoKoot** [lukemelas/EfficientNet-PyTorch#280]
Hey! Is AutoAugment somehow built into each network or do you only separately apply it in your pre-training? I am using your efficientnet implementation in my own training environment to compare it with other models for a research paper, but am unsure whether the efficientnet I am using from this implementation gets an unfair advantage because it has additional augmentations built in under the hood. If you can let me know that would be great! Thanks!

**Comment by lukemelas** [comment]
Hi @RaivoKoot, AutoAugment is not built into the network -- it's just used for training. Based on what you've said, I think it will be a fair comparison.

**Comment by lukemelas** [comment]
Closing this issue, feel free to reopen if you have more questions

**Comment by RaivoKoot** [comment]
Aww that is amazing. Thanks for the quick reply!
